### ISSUE TITLE
Sort nodes by status

### ISSUE THREAD

**Issue by zaelgohary** [threefoldtech/tfgrid-sdk-ts#1786]
Blocked on [this](https://github.com/threefoldtech/tfgrid-sdk-go/issues/566)

**Comment by zaelgohary** [comment]
Already done [here](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2197#pullrequestreview-1971046637)

**Comment by A-Harby** [comment]
Verified, Devnet 5c12959.

The `Select node status` filter is added to the node finder.
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/f7104b87-b72a-4847-8413-ec2fa145ca1d)
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/29330498-46e2-4c82-a05a-647b79ed2c30)

[TC2255 - Filter Online/Offline/Standby nodes](https://app.testlodge.com/a/26076/projects/40893/suites/234374?expand_section=426661#case_4150464)

**Comment by ramezsaeed** [comment]
Verified:
- Devnet
- 5c12959
